geomesa-ui/src/main/ui/src/app/map/map.js

```diff
@@ -42,6 +46,20 @@ angular.module('geomesa.map', [])
                     }
                 };
 
+                scope.map.on('singleclick', function(evt) {
+                    var viewResolution = olView.getResolution();
```

Comment:
should only have one "var" keywords per scope and it should be on the first line.